rules-reviewed/azure/eap/test/eap-to-azure-appservice-environment-variables.windup.test.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<ruletest xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+          id="eap-to-azure-appservice-environment-variables-test" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <testDataPath>data/eap-to-azure-appservice-environment-variables</testDataPath>
+    <!-- By default sourceMode is True, so it will only consider .java files, not .jar binaries -->
+    <sourceMode>true</sourceMode>
+    <rulePath>../eap-to-azure-appservice-environment-variables.windup.xml</rulePath>
+    <ruleset>
+        <rules>
+            <rule id="eap-to-azure-appservice-environment-variables-001-test">
+                <when>
+                    <not>
+                        <iterable-filter size="3">

```

Comment:
Once the `filecontent` conditions will be removed, the test can have `size=2` as expected because there are just 2 matching statement in the `AppServiceSystemGetEnvGetProperty.java` test file.
```suggestion
                        <iterable-filter size="2">

```

Reply:
Updated